lib/ruby_lsp/server.rb

```diff
@@ -104,7 +104,7 @@ def load_addons
           ),
         )
 
-        $stderr.puts(errored_addons.map(&:backtraces).join("\n\n"))
+        $stderr.puts(errored_addons.map(&:errors_details).join("\n\n"))
```

Comment:
When testing with a `raise` on the first like of `activate` in an addon, I found that the Notification isn't displaying. I'm not entirely sure why that is yet. But I think it's worth writing the error details alongside the backtrace in stderr.